### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: include/deal.II/lac/trilinos_sparse_matrix.h [hunk 1]
```diff
@@ -2403,7 +2403,7 @@ namespace TrilinosWrappers
          * Epetra_Operator.
          *
          * @note This function will only be operable if the payload has been
-         * initalised with an InverseOperator, or is a wrapper to a preconditioner.
+         * initialised with an InverseOperator, or is a wrapper to a preconditioner.
```
Comment (davydden) [inline include/deal.II/lac/trilinos_sparse_matrix.h]: would you mind making this `initialized` ?

**Comment by bangerth** [comment]
Oh, how nice! Great work @luzpaz, these efforts are really appreciated!

File: include/deal.II/lac/trilinos_sparse_matrix.h (hunk 1 above)
Comment (luzpaz) [reply include/deal.II/lac/trilinos_sparse_matrix.h]: No problem. Will do that and repush  soon

**Comment by luzpaz** [comment]
Please also examine:  
0abd4ea073dbf991c4ed683e7988d2a4399ee5cd  
af98d6ba80082577c2ecd476ccfe50ef82f95aa1

**Comment by drwells** [comment]
Thanks again for doing this, and also for (I checked out the codespell repo) writing a significant part of this excellent tool :+1:

**Pull request merged**